### ISSUE TITLE
Cannot navigate to play controls while using a screen-reader.

### ISSUE THREAD

**Issue by Murcielago7384** [audacious-media-player/audacious#1830]
**Describe the bug**
I use NVDA, which is a free and open source screen-reader for Windows available from NV Access. When Audacious is open with a playlist playing, the only keyboard navigation I can do is using up/down arrows to move through the playlist. When using a screen-reader, pressing tab is a common method to move through controls on a web page or app. However, pressing tab does nothing.

**Steps to reproduce**
For this bug, you may need to install NVDA. Have a playlist open and try pressing tab. The music does not need to be playing.

**Expected behavior**
As stated above, pressing tab is a common method to cycle through controls on a webpage/application. Ideally, I should be able to tab to the media control buttons inside Audacious ('play' ETC). However, there is no way to navigate outside of the playlist's tracklist.
 
**Additional information**
NVDA is free, and open source. Their site contains guides on how to make applications accessible, should this be needed. I am very grateful that Audacious includes keyboard shortcuts, but it would still be nice to be able to access controls without having to memorise all the necessary shortcuts.

* Audacious version and interface (Qt / GTK / Winamp)
Version 4.5.1, Qt interface. I know 4.6.1 is coming to Windows soon, so I just wanted to put this on your radar for a future release.

* Operating system and version
Windows 11 25H2.
